features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/RumConfiguration.kt

```diff
@@ -257,6 +259,17 @@ data class RumConfiguration internal constructor(
             return this
         }
 
+        /**
+         * Sets the identifier strategy for initial network resources used to compute the time to network settled
+         * in a RUM View event. By default, the SDK uses a [TimeBasedInitialResourceIdentifier] with
+         * a threshold of 100ms.
+         * @param initialResourceIdentifier the [InitialResourceIdentifier] to use.
+         */
+        fun setNetworkSettledInitialResourceIdentifier(initialResourceIdentifier: InitialResourceIdentifier): Builder {
```

Comment:
> [!NOTE]
> This function's name is a bit long, maybe we can rename it to `setInitialResourceIdentifier`?

Reply:
Glad that you mentioned it...was also thinking about it but I was afraid that only using `setInitialResourceIdentifier` will not be explicit enough. I guess we are ok though.